src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -189,14 +189,23 @@ private void RetrieveMatchingProcessesByProcessName()
                 if (!found &&
                     !WildcardPattern.ContainsWildcardCharacters(pattern))
                 {
+                    string errorText = ProcessResources.NoProcessFoundForGivenName;
+                    string errorName = "NoProcessFoundForGivenName";
```

Comment:
Ah, we need use `nameof()`. Below too.

Reply:
I thought it looked like it needed something!

Reply:
```csharp
nameof(ProcessResources.NoProcessFoundForGivenName)
```

gives you "NoProcessFoundForGivenName" and not "ProcessResources.NoProcessFoundForGivenName"?

Neat. I didn't know that. `nameof` is so good.